galley/pkg/config/source/kube/rt/known.go

```diff
@@ -162,6 +167,21 @@ func (p *Provider) initKnownAdapters() {
 				}
 				return out, nil
 			},
+			isEqual: func(o1 interface{}, o2 interface{}) bool {
```

Comment:
was this code block accidentally omitted in the previous PR?

Reply:
I was pruning the package as I was refactoring. I didn't need equal at the time, so I removed it. When implementing the apiserver package, I needed it again so had to add it again.